packages/hash/api/src/model/knowledge/entity.model.ts

```diff
@@ -132,13 +153,15 @@ export default class {
     const cachedEntityTypeModels = new Map<string, EntityTypeModel>();
 
     return await Promise.all(
-      entities.map((entity) =>
-        EntityModel.fromPersistedEntity(
-          graphApi,
-          entity,
-          cachedEntityTypeModels,
+      entities
+        .filter(({ identifier }) => identifier.ownedById === params.accountId)
```

Comment:
Gotcha, but:
> The description of the method just says that it's the account requesting the entities, it doesn't say that it only returns entities owned by that account.

Reply:
> So I'm guessing we'd expect getAllLatest to return all private and public entities that the user has access too, including ones owned by an Org for example?

I don't really understand why we want such a method for entities in the first place. Because of the potential size of this set of entities, a query method seems like the more logical method to call.

Can I remove this method? It's only being used in `User.getUseByAccountId`, and I think I can replace it with a `EntityModel.getbyQuery` method instead.

Reply:
That sounds better to me

Reply:
I've deprecated the method in https://github.com/hashintel/hash/pull/1081/commits/26bb3046b874928220f1859bb97017aa50f7cef8